lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -170,6 +170,11 @@ def gallery(self) -> Optional[AzureVmGallerySchema]:
         gallery: Optional[AzureVmGallerySchema] = self._gallery
         if not gallery:
             if isinstance(self.gallery_raw, dict):
+                # no impact for deployment, make the info consistent
```

Comment:
try to be as clear as possible. The below line contains several operations, but we need to explain to lower here. So the comment should be like,

`Users decide the cases of image names, and the inconsistent cases causes the mismatched error in notifiers. The lower() normalizes the image names, it has no impact on deployment.`